packages/deployments/contracts/src/cli/init/helpers/assets.ts

```diff
@@ -29,15 +29,6 @@ export const setupAsset = async (args: {
     `\n\tVerifying asset setup for ${asset.name} (${asset.canonical.address}). Canonical ID: ${canonical.id}; Canonical Domain: ${canonical.domain}; Key: ${key}`,
   );
 
-  // Set up the canonical asset on the canonical domain.
```

Comment:
This setup script needs to be able to register xERC20s using the pseudo-canonical domain as well as maintain the ability to register other assets that are homed on Eth. I don't think we can just remove the home checks, we should still have those guards. 

We need to separate logic based on whether `11111` is specifically set as the canonical domain for an asset. If not, then the original checks should still apply.